services/QuillLMS/client/app/entrypoints/snippets/translated_text.ts

```diff
@@ -0,0 +1,78 @@
+function adjustTextareaHeight(textarea) {
```

Comment:
I don't object to skipping React for this, but wonder how the front-end folks (@eadams17, @emilia-friedberg) feel about it.  Not sure if we want to be long-term committed to supporting this, but maybe it's a reasonable short-term solution and if we ever need to enhance or fix it we'd just replace it with something more robust.

Reply:
@anathomical yeah, I don't think this is a pattern we want to follow in the future -- I don't really like the idea of having a bunch of vanilla Javascript files floating around, which force us to do things like the native implementation of fetch Samantha has here rather than the shared `requestX` functions we use elsewhere -- but as effectively a prototype I think it's okay given the short timetable Samantha has for this project. My preference would be to rewrite it whenever we next need to make a change.

Reply:
sounds good. Something I've been thinking is that the react app is actually quite separate from the backend and could even eventually be taken out of rails and deployed on its own. I think there might be a few advantages to that, and we could simplify our pipeline. But obviously a much longer-term project that we'd have to discuss as a team.